.github/workflows/build.yaml

```diff
@@ -8,14 +8,14 @@ on: ["push", "pull_request"]
 jobs:
   lint:
     runs-on: ubuntu-latest
-    container: erlang:26
+    container: erlang:28
     steps:
       - name: Checkout
         uses: actions/checkout@v4
       - name: Install erlfmt
         run: |
           cd ..
-          git clone --depth 1 -b v1.1.0 https://github.com/WhatsApp/erlfmt.git
+          git clone --depth 1 -b v1.6.2 https://github.com/WhatsApp/erlfmt.git
```

Comment:
I wonder if it would be simpler for you to maintain by just adding erlfmt as a rebar3 plugin under a profile in rebar.config? That way you don't need to download erlfmt from github and build the escript explicitly. Then this is as simple as `rebar3 as check fmt -c`